### ISSUE TITLE
About Data Generation

### ISSUE THREAD

**Issue by JaneFo** [Dekai21/Multi_Agent_Intersection#1]
When I generate the training set and validation set from SUMO, the track is []. Looking forward to your solution.

**Comment by Dekai21** [comment]
Hi, thanks for your interest in this project! I am currently checking the code by running it on a new machine, will let you know once I finish checking the code of data generation.

**Comment by JaneFo** [comment]
您好，我是付峥，来信我已收到，我会在查收邮件之后给您回复。

**Comment by Dekai21** [comment]
Hi, I ran the code on a new machine and realized that some scripts and files are missing. Now I added the necessary scripts and files, and the data generation should work now. Feel free to test it again.

**Comment by JaneFo** [comment]
您好，我是付峥，来信我已收到，我会在查收邮件之后给您回复。
